operations/flags.go

```diff
@@ -230,7 +230,7 @@ func addDbSettingsFlags(flags ...cli.Flag) []cli.Flag {
 func addRefFlag(flags ...cli.Flag) []cli.Flag {
 	return append(flags, cli.StringFlag{
 		Name:  refFlagName,
-		Usage: "diff with `REF`",
+		Usage: "specify a commit on top of the merge-base to test",
```

Comment:
If i'm interpreting the flag correctly, the commit has to be some commit between the merge-base and the HEAD of the current checked out branch?

I think it's hard to communicate this in the message, so this looks LGTM!